setup.cfg

```diff
@@ -96,6 +96,7 @@ install_requires =
     flask-appbuilder~=3.4, <4.0.0
     flask-caching>=1.5.0, <2.0.0
     flask-login>=0.3, <0.5
+    flask-session>=0.3.2
```

Comment:
As it is now, I'm tempted to pin this to 0.4.0, as we'd need a migration for any schema changes in the model anyways.

Reply:
Yeah, I'd say pin this - don't forget a comment saying why we have done it

Reply:
Pretty much same difference, but I set the upper bound at the latest release instead of pinning to just the latest version.

(oh, and fixed the lower bound that I messed up originally 🙃)